### DISCUSSION TITLE
Catholic moral theology dilemma.

### DISCUSSION THREAD

**Discussion by JosephMLu** [xai-org/grok-1#310] [General]

**Comment by Lord-Highfixer** [comment]
Sola Scriptura. By scripture alone.

1. AI prompts should pull answers closer to Word-for-Word
2. Understand the difference between Prescriptive and Descriptive Bible verses since so many don't understand the difference
3. Give insight into the true words used in Greek, Hebrew, and Aramaic as the Bible was written in those original languages. King James with Strong's for example
4. Zero wokeism, zero leftism, sola scriptura
